cromshell

```diff
@@ -1400,36 +1472,44 @@ function setupCromshell()
 # The bash equivalent to `if __name__ == '__main__':
 if ${ISINTERACTIVESHELL} ; then
 
-  # Check the remaining arguments for help and display it if we have to: 
+  # Check the remaining arguments for help and display it if we have to:
   checkForComplexHelpArgument $@
 
   # Get flags:
-  while getopts ":t:" opt ; do
-    case ${opt} in 
+  while getopts ":t:xp" opt ; do
+    case ${opt} in
       t)
         # Ensure that the value given to -t is numerical:
-        echo ${OPTARG} | grep -q -E '^[0-9]+$' 
+        echo ${OPTARG} | grep -q -E '^[0-9]+$'
         rv=$?
         [ $rv -ne 0 ] && invalidSubCommand '' "flag value (${OPTARG})" "-${opt} argument must be an integer."
 
         # Set our timeouts:
         CURL_CONNECT_TIMEOUT=${OPTARG}
         let CURL_MAX_TIMEOUT=2*${CURL_CONNECT_TIMEOUT}
         ;;
+      x)
```

Comment:
Since these aren't global options I think they should be moved into the `execution-status-count` function and handled there.

It would work largely the same way, but you can reference `list` for a good example.

Reply:
This is going to be a little more complicated since we want `execution-status-count` to still run on the implicit workflow IDs, but there's not current mechanism in the code to do that AND handle subcommand-specific args.